manifests/plugin/mysql.pp

```diff
@@ -4,6 +4,7 @@
   $ensure           = present,
   $manage_package   = $collectd::manage_package,
   $interval         = undef,
+  $databases        = { },
```

Comment:
Can we set this to `false` or `undef` and wrap the call to `create_resources` in an `if` statement? We can use `validate_hash` to ensure that `$databases` is a hash.